tests/test_message_template.py

```
@@ -389,7 +389,12 @@ class TestBetsMessagesPlannatechErrorTypes:
     InsufficientBalance reuses the existing `without_funds` field; these cover
     the remaining business errorTypes. Each is `Optional[MessageItem] = None`."""
 
-    NEW_FIELDS = ["market_unavailable", "bet_limit_exceeded", "bet_amount_too_low"]
+    NEW_FIELDS = [
+        "market_unavailable",
+        "bet_limit_exceeded",
+        "bet_amount_too_low",
+        "minimum_potential_winning",
+    ]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)